loopout.go

```diff
@@ -1557,7 +1559,14 @@ func (s *loopOutSwap) sweepMuSig2(ctx context.Context,
 		labels.LoopOutSweepSuccess(swap.ShortHash(&s.hash)),
 	)
 	if err != nil {
-		s.log.Warnf("Publish of MuSig2 sweep failed: %v", err)
+		var sweepTxBuf bytes.Buffer
+		if err := sweepTx.Serialize(&sweepTxBuf); err != nil {
+			s.log.Warnf("Unable to serialize sweep txn: %v", err)
+		}
+
+		s.log.Warnf("Publish of MuSig2 sweep failed: %v. Raw tx: %s",
```

Comment:
nit: can use `%x` for hex encoding directly.

Reply:
done